Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM golang:alpine AS build
 RUN apk add --no-cache git
+RUN apk add --no-cache tzdata
```

Comment:
This is the build container for the gphotos-cdp app. You need to add it to the runtime container below.